test/extended/router/stress.go

```diff
@@ -78,6 +80,23 @@ var _ = g.Describe("[sig-network][Feature:Router][apigroup:route.openshift.io]",
 				Name: "system:router",
 			},
 		}, metav1.CreateOptions{})
+		// The router typically runs with allowPrivilegeEscalation enabled; however, system:router defaults to the
+		// restricted-v2 scc, which disallows privilege escalation. The restricted policy permits privilege escalation.
```

Comment:
I don't think we need to keep this comment as it may mislead. We just want this router's POD to be assigned to `restricted` SCC as CIO managed.

Reply:
```suggestion
		// The router typically runs with allowPrivilegeEscalation enabled; however, all service accounts are assigned to
		// restricted-v2 scc by default, which disallows privilege escalation. The restricted policy permits privilege escalation.
```

Reply:
Done.